packages/frontend/src/preview/data.ts

```diff
@@ -62,9 +63,11 @@ const createFakeOffers = (count: number): ForcedTradeOfferEntry[] =>
       (amountCollateral * BigInt(10 ** AssetId.decimals(assetId))) /
       amountSynthetic /
       10_000n
+    const createdAt = Timestamp(Date.now() - 50000 - i * 480 * 3570)
```

Comment:
Just out of curiosity - is there any reasoning for these numbers?

Reply:
Just wanted to see something that's not regularly every 30 minutes